src/strategies/erc721-with-tokenid-range-weights-simple/index.ts

```diff
@@ -1,15 +1,35 @@
+import { strategy as erc721WithMultiplier } from '../erc721-with-multiplier';
 import { multicall } from '../../utils';
+import { WeightRange } from './types';
 
-export const author = 'gregegan';
-export const version = '0.1.0';
+export const author = 'FeSens';
+export const version = '0.2.0';
 
 const abi = [
-  'function ownerOf(uint256 tokenId) public view returns (address owner)'
+  'function ownerOf(uint256 tokenId) public view returns (address owner)',
+  'function balanceOf(address account) external view returns (uint256)'
```

Comment:
Why balanceOf ? can't see where you used it?

Reply:
Ops, my bad. It was used before I used `erc721WithMultiplier` inside this strategy.